README.md

```diff
@@ -46,15 +47,15 @@ None
 
 | Variable | Default |
 |----------|---------|
-| `__java_packages` | `["java-1.8.0-openjdk"]` |
+| `__java_packages` | `["java-1.8.0-openjdk", "java-1.8.0-openjdk-devel"]` |
 
-## Ubuntu-14.04
+## Ubuntu-14
```

Comment:
| versions in README.md | versions in meta/main.yml | OK/NG | explanation |
|--------------------------------|----------------------------------|-----------|---------------|
| N/A | some versions | OK | because it is obvious that versions described in meta/main.yml are supported versions |
| some versions described in meta/main.yml | some versions | OK | because they matched and no conflict |
| some versions | N/A | NG | because meta/main.yml must have supported versions |
| some versions NOT matched with those in meta/main.yml | some versions | NG | because users could not know which versions are supported |

i do not think it is a good idea to send users a message that the role might work for non-supported versions via README.md as we do not test it at all.